src/tinyspline.h

```diff
@@ -5,6 +5,22 @@
 
 #include <stddef.h>
 
+#ifdef _WIN32
+# define TINYSPLINE_SHARED_EXPORT __declspec(dllexport)
```

Comment:
Usually, I do not indent definitions in conditions. Is this convention?

Reply:
You're the owner of tinyspline, so you make the rules.
I just thought it was more readable.
But I'll change it if you want.

Reply:
I prefer consistency :). Thanks.